Services/Mail/classes/class.ilMailOptions.php

```diff
@@ -109,13 +120,12 @@ public function createMailOptionsEntry()
 		);
 	}
 
-	/**
-	 * 
-	 */
 	protected function read()
 	{
 		$res = $this->db->queryF(
-			'SELECT * FROM ' . $this->table_mail_options . ' WHERE user_id = %s',
+			'SELECT * FROM mail_options 
```

Comment:
We should only select '*' from 'mail_options' as well as 'email' and 'second_email' from 'usr_data.